data/campaigns/The_Hammer_of_Thursagan/scenarios/08_The_Siege_of_Kal_Kartha.cfg

```diff
@@ -0,0 +1,371 @@
+    #############################
+    # DULCATULOS DIES
+    #############################
+    [event]
+        name=die
+        [filter]
+            id=Dulcatulos
+        [/filter]
+        [message]
+            speaker=Aiglondur
+            message= _ "Kal Kartha is taken. Our mission has failed."
+        [/message]
+        [endlevel]
+            result=defeat
+        [/endlevel]
+    [/event]
+
+    #############################
+    # SECOND ORC LEADER DIES
+    #############################
+    [event]
+        name=die
+        [filter]
+            id=leader3,leader4,leader5,leader6
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                id=leader3,leader4,leader5,leader6
+                count=0-2
+            [/have_unit]
+        [/filter_condition]
+        [message]
+            id=leader3,leader4,leader5,leader6
+            formula=( self.hitpoints>0 )
+            message=_"Useless grunts! We should have taken the gate weeks ago! Fight harder!"
+        [/message]
+        [gold]
+            amount={ON_DIFFICULTY4 10 20 30 40}
+            side=3,4,5,6
+        [/gold]
+        [modify_side]
+            side=3,4,5,6
+            income={ON_DIFFICULTY4 0 5 10 15}
+        [/modify_side]
+    [/event]
+
+    #############################
+    # ORC LEADERS DIE
+    #############################
+    [event]
+        name=die
+        [filter]
+            id=leader3,leader4,leader5,leader6
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                id=leader3,leader4,leader5,leader6
+                count=0-1
+            [/have_unit]
+        [/filter_condition]
+
+        #--------------------
+        # ORCS FLEE
+        #--------------------
+        [message]
+            id=leader3,leader4,leader5,leader6
+            formula=( self.hitpoints>0 )
+            message=_"Grr... nasty stinky-midgets. We know what you’re up to in there. We’ll be back."
+        [/message]
+        {CUTSCENE_ENEMIES_RETREAT 3  40 "$($this_item.y)"}
+        {CUTSCENE_ENEMIES_RETREAT 4  "$($this_item.x)" 1}
+        {CUTSCENE_ENEMIES_RETREAT 6  40 "$($this_item.y)"}
+        {KILL side=5,7}
+
+        #--------------------
+        # DULCATULOS SPEAKS
+        #--------------------
+        {MOVE_UNIT id=Dulcatulos 36 3}
+        {MOVE_UNIT id=Dulcatulos 36 4}
+        [message]
+            speaker=Dulcatulos
+            message=_"Hail, fellow dwarves! Well fought, and our thanks; we were sore pressed."
+        [/message]
+        [message]
+            speaker=Dulcatulos
+            message=_"I had not expected to find allies so far from peopled lands. From whence do you hail?"
+        [/message]
+        [message]
+            speaker=Aiglondur
+            message= _ "We come at the behest of Hamel, King of Knalga and Lord Companion of the Northern Alliance. We seek audience with your lord."
```

Comment:
Is Hamel a lord or a king? It should be consistent with his mention in S2, regardless.